src/pages/index.tsx

```diff
@@ -92,6 +94,12 @@ export default function Page() {
   const { t } = useTranslation('common')
 
   const { dialogProps } = useVerificationOAuthHandler()
+  const { trackRegistrationEvent } = useRegistrationEventTracker()
+
+  useEffect(() => {
+    trackRegistrationEvent('Home page')
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
 
```

Comment:
I think this is why you set strictMode to false. In dev mode it will run twice but in production it will only run once. This is okay.